src/components/organisms/AssetActions/Trade/Output.tsx

```diff
@@ -0,0 +1,70 @@
+import { useOcean } from '@oceanprotocol/react'
+import { FormikContextType, useFormikContext } from 'formik'
+import React, { ReactElement, useEffect, useState } from 'react'
+import { TradeLiquidity } from '.'
+import Token from '../Pool/Token'
+import styles from './Output.module.css'
+
+export default function Output({
+  dtSymbol,
+  poolAddress
+}: {
+  dtSymbol: string
+  poolAddress: string
+}): ReactElement {
+  const { ocean } = useOcean()
+  const [minOutput, setMinOutput] = useState<string>()
+  const [swapFee, setSwapFee] = useState<string>()
+
+  // Connect with form
+  const { values }: FormikContextType<TradeLiquidity> = useFormikContext()
+
+  // Get swap fee
+  useEffect(() => {
+    if (!ocean || !poolAddress) return
+
+    async function getSwapFee() {
+      const swapFee = await ocean.pool.getSwapFee(poolAddress)
+      // swapFee is tricky: to get 0.1% you need to convert from 0.001
+      setSwapFee(`${Number(swapFee) * 100}`)
+    }
+    getSwapFee()
+  }, [ocean, poolAddress])
+
+  // Get output values
+  useEffect(() => {
+    if (!ocean || !poolAddress) return
+
+    async function getOutput() {
+      // Minimum received
+      // TODO: check ity is actually the minimum received after transaction.
+      const minOutput =
+        values.type === 'buy'
+          ? await ocean.pool.getDTReceived(poolAddress, `${values.ocean}`)
```

Comment:
might be redundant fetching this here since the output from this seems to be always the same as whatever is put into the bottom input field.

Reply:
Yes, we should put something like 'Maximum price'  since both buy and sell have a fixed amount and a variable price. The slippage is hardcoded to 10% for now, we can't actually calculate it